src/adapter/activateDaffodilDebug.ts

```diff
@@ -24,6 +24,35 @@ import * as launchWizard from '../launchWizard/launchWizard'
 import * as omegaEditClient from '../omega_edit/client'
 import * as dfdlLang from '../language/dfdl'
 
+// Method to file path for program and data
```

Comment:
Is there any sort of javadoc-like thing for typescript?

Reply:
Yeah, I just googled that and found http://typedoc.org/example/ (among some other options).  Would be good to integrate something, and I can do the same on the Ωedit side since the TypeScript client there is a "public facing" API.  Scala has ScalaDoc, and I'm using Doxygen for the C and C++ API.